src/main/java/org/opentestsystem/ap/ims/rest/VersionedItemApi.java

```diff
@@ -0,0 +1,54 @@
+package org.opentestsystem.ap.ims.rest;
+
+import lombok.extern.slf4j.Slf4j;
+import org.opentestsystem.ap.common.model.Item;
+import org.opentestsystem.ap.ims.model.ItemResource;
+import org.opentestsystem.ap.ims.service.VersionedItemService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.CacheControl;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v1/versioned-items")
+public class VersionedItemApi {
+    private final VersionedItemService versionedItemService;
+
+    @Autowired
+    public VersionedItemApi(final VersionedItemService versionedItemService) {
+        this.versionedItemService = versionedItemService;
+    }
+
+    @GetMapping("/{itemId}")
+    public Item getItem(@PathVariable final String itemId,
+                        @RequestParam final String version) {
```

Comment:
doesn't this have to accept just the "opaque" (or "fully qualified") item ID? the client shouldn't have to know it needs to send the version part of the ID as a query param right? Or is this endpoint to service something else?

Reply:
Item Translation Service will be the one accepting the "opaque" id. It will parse it and call this services with the parsed data. Sorry for keeping you out of the loop, there were a few discussion about it today going back and forth. We have asked Brett to help us with the TIMS side of the code to make sure we do not miss anything.

Reply:
got it - thanks for looping me in : j
If that is the case this PR looks good as far as i can see 👍